databuilder/file_formats/__init__.py

```diff
@@ -47,3 +63,11 @@ def get_file_extension(filename):
         return "".join(filename.suffixes[-2:])
     else:
         return filename.suffix
+
+
+def get_methods(extension):
+    if extension == ".feather":
+        extension = ".arrow"
```

Comment:
I think we were planning to use `.arrow` consistently as that seems to be the de-facto standard now. It's quite possible some of the old Github issues still talk about `.feather` though, in which case: apologies!

Reply:
The current docs use `.feather` (e.g. https://docs.opensafely.org/actions-scripts/#large-highly_sensitive-output-files https://docs.opensafely.org/measures/#calculate-the-measures) so I figured I'd cover both bases.

Reply:
Yeah, it's true that's what Cohort Extractor uses. But I think we're going for a clean break, fix-everything-at-once approach with ehrQL so I'd avoid carrying over legacy stuff unless we've got a demonstrable need for it.

Reply:
That seems fair, I'll yank out `.feather` and remove that method